ci/params/jvm-support-opts/quickstart-bitbucket-jvm-support-opts.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "ParameterKey": "DBMultiAZ",
+    "ParameterValue": "false"
+  },
+  {
+    "ParameterKey": "DBMasterUserPassword",
+    "ParameterValue": "$[taskcat_genpass_8S]"
+  },
+  {
+    "ParameterKey": "BitbucketAdminPassword",
+    "ParameterValue": ""
+  },
+  {
+    "ParameterKey": "CidrBlock",
+    "ParameterValue": "0.0.0.0/0"
+  },
+  {
+    "ParameterKey": "DBIops",
+    "ParameterValue": "1000"
+  },
+  {
+    "ParameterKey": "DBPassword",
+    "ParameterValue": "$[taskcat_genpass_8S]"
+  },
+  {
+    "ParameterKey": "KeyPairName",
+    "ParameterValue": "replaced-by-taskcat-override-file"
+  },
+  {
+    "ParameterKey": "ClusterNodeInstanceType",
+    "ParameterValue": "t3.medium"
+  },
+  {
+    "ParameterKey": "DBInstanceClass",
+    "ParameterValue": "db.t3.medium"
+  },
+  {
+    "ParameterKey": "QSS3BucketName",
+    "ParameterValue": "$[taskcat_autobucket]"
+  },
+  {
+    "ParameterKey": "QSS3KeyPrefix",
+    "ParameterValue": "quickstart-atlassian-bitbucket/"
+  },
+  {
+    "ParameterKey": "JvmSupportOpts",
+    "ParameterValue": "-Dhttp.proxyHost=10.15.112.40 -Dhttp.proxyPort=3128 -Dhttps.proxyHost=10.15.112.40 -Dhttps.proxyPort=3128 -Dhttp.nonProxyHosts=localhost|169.254.169.254"
```

Comment:
When using these specific IP addresses, does the Bitbucket starts and it is accessible?

Reply:
Yes, I confirm Bitbucket.service is loaded, started and running.